spring-boot-examples/openfeign-app/src/main/java/io/dapr/springboot/examples/openfeign/ProducerClientRestController.java

```diff
@@ -0,0 +1,34 @@
+package io.dapr.springboot.examples.openfeign;
```

Comment:
Hi @lony2003 - Appreciate the thorough PR :) Please ensure all files added have the header at the top. I've triggered CI to run. Lets ensure its green and Ill do a thorough pass at the diff

Reply:
Resolved.